### ISSUE TITLE
Create Photoshop Mockup

### ISSUE THREAD

**Issue by RealEvanPonter** [JonLeeCon/GetSteady#6]
Create a mockup of what the layout should look like

**Comment by RealEvanPonter** [comment]
Hey, I worked up a mockup for the mobile site. Mobile-first, right?
![getsteadymusic mockup](https://cloud.githubusercontent.com/assets/11358831/24086572/11dc5da4-0ce8-11e7-8320-d179fbc82fed.png)
 Let me know what you think!

**Comment by JonLeeCon** [comment]
Do you have a psd that I can download? I am writing down some minor things, but definitely a good starting point

**Comment by RealEvanPonter** [comment]
Yup, but GitHub doesn't support .ai as an attachment, I'll put it on our Google Drive [here](https://drive.google.com/open?id=0B1xhk4oO-x-GWUJySHZ4OHk3Q2s)
